src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,43 @@
+$color-light-blue: rgba(0, 172, 220, 1);
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: inline-block;
+    text-decoration: none;
+    font-size: 12px;
+    text-transform: uppercase;
+    font-weight: 500;
+    line-height: 60px;
+    color: rgba(6, 11, 53, 1);
+    transition: all 0.3s ease-in-out;
```

Comment:
don't use it like that, because it's the same as transition-duration: 0.3s
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/222679244-9d82158c-e7e2-4caf-b10c-8d76dbb23d72.png)
Also, create variable for transition duration
```suggestion
    transition: color $transitionDuration;
```